packages/dashboard/src/CdcDashboard.tsx

```diff
@@ -6,6 +6,7 @@ import defaults from './data/initial-state'
 import { processData } from './helpers/processData'
 import { getVizKeys } from './helpers/getVizKeys'
 import { processDataLegacy } from './helpers/processDataLegacy'
+import { addValuesToSharedFilters } from './helpers/addValuesToSharedFilters'
```

Comment:
can we merge dev-8191 before this ticket? I've moved the addValuesToSharedFilters function. Also in packages/core/components/Filters.tsx in that PR I have added this addValuesToSharedFilters function so maybe that will resolve this bug?